### PR TITLE
feat(content): change CCOR planet descriptions to use "cosmodromo" instead of "spaceport"

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Content ( Planet Descriptions )**

This PR addresses the feature described on Discord.

## Summary
All mentions of "spaceport" in CCOR/former CCOR planet and station descriptions have been changed to "cosmodromo" (Portuguese for cosmodrome), to fit in better with lore.

Should there be a mission that tells you the CCOR uses cosmodromo instead of spaceport?

### CODE REVIEW

**Comment by Ebreus** [comment]
Thanks for putting together a PR!
Continuing on from the conversation in Discord, I do think 'Cosmodromo' fits a little better even if we can't include the accent.

**Comment by Zireael07** [comment]
isn't the English spelling cosmodromE with an E?

**Comment by TheGiraffe3** [comment]
> In this case there should be a translator comment so that future translators do not get confused

By "translator comment" in this case do you mean a commented out line detailing how the CCOR uses Portuguese? I believe that's fairly common knowledge.
~~or is it just that I read the pirate campaign PR~~

File: data/map planets.txt [hunk 1]
```diff
@@ -165,7 +165,7 @@ planet Alvorada
...
-	spaceport `Alvorada's spaceport was built into one of the larger craters. Everything about the installation here is as fortified as possible - various hangars for even heavy warships are dug into the walls. From the ground, you can see where artillery pieces were stationed, obscured from above but with clear firing lines at incoming vessels. Now that you've seen them, you can think of hundreds of places across the asteroid where similar emplacements could have been hidden.`
+	spaceport `Alvorada's cosmodromo was built into one of the larger craters. Everything about the installation here is as fortified as possible - various hangars for even heavy warships are dug into the walls. From the ground, you can see where artillery pieces were stationed, obscured from above but with clear firing lines at incoming vessels. Now that you've seen them, you can think of hundreds of places across the asteroid where similar emplacements could have been hidden.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:168]:
```suggestion
	spaceport `Alvorada's cosmodromo was built into one of the larger craters. Everything about the installation here is as fortified as possible - various hangars for even heavy warships are dug into the walls. From the ground, you can see where artillery pieces were stationed, obscured from above but with clear firing lines at incoming vessels. Now that you've seen them, you can think of hundreds of places across the asteroid where similar emplacements could have been hidden.`
	# NOTE: "cosmodromo" is intentional. The CCOR uses Portuguese, and "cosmodromo" is the Portuguese spelling of "cosmodrome".
```

**Comment by Tangle10** [comment]
I honestly don't think this is particularly fitting for the CCOR, unless they're openly trying to invoke the USSR. The USSR having Cosmonauts versus the USA's Astronauts and the like are quite literally ancient history, a long-dead ideological conflict- it'd be like a modern polity stating that it is a descendant of the Crusader states and that it intends to retake Jerusalem, something that immediately consigns it to a ridiculous political fringe. Unless the term is part of standard language in the region for some _other_ reason, I'm personally inclined to say this doesn't feel like an apt change.

**Comment by Zireael07** [comment]
"Cosmonauts" is a still (2024) used term in most of Central Europe. It is not in any way limited to USSR

I do not think having varying terms is a bad idea. Quite the opposite.

Also: see China's taikonauts

**Comment by bene-dictator** [comment]
I'm not convinced. Spaceport descriptions are written from the perspective of the player, and I don't see why the player character would change how they would term a spaceport generally, just because they're in the CCOR– unless they referred to the proper name of the spaceport, then I'd describe the full name of the spaceport. e.g. on Wikipedia, the article for Baikonur is titled "Baikonur Cosmodrome" but it is called a spaceport in the very first line.

In dialogue, it's probably fine to refer it as a cosmodrome/cosmodromo, but I imagine that use would be limited to people from the CCOR and not the player character.

**Comment by Zireael07** [comment]
> but I imagine that use would be limited to people from the CCOR and not the player character.

True, I missed the fact that it is the player character using the term

**Comment by Ebreus** [comment]
> The USSR having Cosmonauts versus the USA's Astronauts and the like are quite literally ancient history, a long-dead ideological conflict- it'd be like a modern polity stating that it is a descendant of the Crusader states and that it intends to retake Jerusalem

I'm starting to reconsider my support for the term however CCOR existing with the history it has contradicts the notion that the ideological conflict is long-dead. CCOR existed as a socialist state within living memory of the start of the game and there is an active movement for its restoration.

**Comment by Ebreus** [comment]
> "Cosmonauts" is a still (2024) used term in most of Central Europe. It is not in any way limited to USSR
> 
> I do not think having varying terms is a bad idea. Quite the opposite.

I agree, I like when alternate terminology used. It's a nice world building tool to show a group is separate from the group you're familiar with. However the more I think about this specifically I'm not sure of it. You said 'Cosmonaut' isn't just Soviet, how 'bout the term for the facility... or is that not common because of the expense of building such facilities?

Considering the history of CCOR I have ... misgivings now that I'm considering this. They weren't originally founded as a socialist state, they had a revolution early in the history of their colony (second generation).

**Comment by Tangle10** [comment]
> > The USSR having Cosmonauts versus the USA's Astronauts and the like are quite literally ancient history, a long-dead ideological conflict- it'd be like a modern polity stating that it is a descendant of the Crusader states and that it intends to retake Jerusalem
> 
> I'm starting to reconsider my support for the term however CCOR existing with the history it has contradicts the notion that the ideological conflict is long-dead. CCOR existed as a socialist state within living memory of the start of the game and there is an active movement for its restoration.

Oh- I mean the ideological conflict specifically between the USSR and the USA, not between socialism and capitalism.

**Pull request closed without merging**